packages/esm-appointments-app/src/helpers/helper.ts

```diff
@@ -23,3 +24,47 @@ export const getServiceCountByAppointmentType = (
     .flat(1)
     .reduce((count, val) => count + val, 0);
 };
+
+function getAppointmentDuration(startTime = 0, endTime = 0) {
+  const diff = endTime - startTime;
+  var minutes = Math.floor(diff / 60000);
+  return minutes + 'Mins';
```

Comment:
We should translate Mins

Reply:
Same here, when I call `useTranslations` it fails unless if I am to pass `t` as a prop which is not a good thing.